courses/machine_learning/deepdive2/feature_engineering/solutions/2_bqml_adv_feat_eng.ipynb

```diff
@@ -0,0 +1,2308 @@
+{
```

Comment:
<p>Drop LO's not unique to the lab:</p><p> </p><ul><li>Code Cleanup</li><li>Create SQL statements to evaluate the model</li><li>Evaluate model performance</li><li>Create a predictive model</li><li>Create an RMSE Summary table</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/777/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
Here are the remaining LO's:

* Perform a feature cross using BigQuery's ML.FEATURE_CROSS
* Derive a coordinate feature
* Feature cross coordinate features
* Apply the BUCKETIZE function
* Apply the TRANSFORM clause
* Apply L2 Regularization
* Evaluate model performance